### PR TITLE
v5

### PR DESCRIPTION
**Pull request by jenius**
A number of breaking changes in here, so this is going to be release v5

- Update all dependencies, get rid of graceful-fs warning on node 6
- Remove `roots deploy`, as ship is no longer maintained
- A variety of other small fixes and cleanup

### CODE REVIEW

**Comment by jenius** [comment]
Note to self, when this is merged turn on greenkeeper

File: lib/api/template.coffee [hunk 1]
```diff
@@ -5,7 +5,7 @@ nodefn        = require 'when/node'
 Sprout        = require '../sprout'
 global_config = require '../global_config'
```
Comment (kylemac) [inline lib/api/template.coffee]: i think you can remove this now, right?

**Comment by kylemac** [comment]
i comment then LGTM. tests pass and i ran it locally on a project 👍

File: lib/api/template.coffee (hunk 1 above)
Comment (jenius) [inline lib/api/template.coffee]: I don't think so, it's used in multiple places in this file

Comment (kylemac) [inline lib/api/template.coffee]: so it is...

**Pull request merged**